app/controllers/bookings_controller.rb

```diff
@@ -1,2 +1,41 @@
 class BookingsController < ApplicationController
+  before_action :find_dress
+
+  def new
+    @booking = Booking.new
+  end
+
+  def create
+    @booking = Booking.new(booking_params)
+    @booking.dress = @dress
+    @booking.user = @current_user
+    if @booking.save
+      redirect_to bookings_path
+      # Is this the path we want to go to?
+    else
+      render :new
+    end
+    # change status?
+  end
+
+  # def destroy
+  #   @booking = Booking.find(params[:id])
+  #   flash[:remove] = "\"#{@dress.brand} dress\" removed from the bookings"
+  #   # flash message to notify dress has been removed from bookings
+  #   @booking.destroy
+  #   redirect_to bookings_path
+  #   # Is this the path we want to go to?
+  # end
+
+  # Commented out destroy method for now!
+
+  private
+
+  def booking_params
+    params.require(:booking).permit(:starting_date, :ending_date, :status)
```

Comment:
If I'm not mistaken, I believe we have to permit :dress_id and :user_id as well. @CamillieFu @KeiranBrown1301 What do you think?

Reply:
I agree - we need dress and user id. 
Also for now i think redirecting to the booking_path should be fine - we may decide to consolidate booking info somewhere like on the user profile later on but for now makes sense to me!

Reply:
I was also conflicted about this because I also put in id’s for watchlist but in other examples Yann would leave it out and I assume it would be given the id and association by active record or by the model. Have to ask Yann about it tonight

Reply:
Yeah, I was just watching today's lecture and it looks like we might not need to permit the user_id after all so Gary is probably correct. Not sure about dress_id but might be the same so could be fine as is :)

Reply:
Ok, let's wait and see what Yann thinks